distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Engine/Default.enso

```diff
@@ -465,7 +465,7 @@ type Pattern
                  pattern = engine.compile "a" []
                  input = "bacadaeaf"
                  pattern.match input
-    split : Text -> Matching_Mode.First | Matching_Mode.Last | Regex_Mode | Integer -> Vector Text
```

Comment:
@radeusgd This is the only implementation of the interface method that was changed in this PR, and it already had both First and Last.